tests/test_sweep_backtest.py

```
@@ -0,0 +1,154 @@
+    assert res.avg_pnl_cents == pytest.approx(-10.0 / 3, 0.01)
+    assert res.max_drawdown_cents == 50.0
+    assert res.profit_factor == pytest.approx(40.0 / 50.0, 0.01)
+
+
+def test_generate_sensitivity_grid():
+    base = BacktestParams(offset=0.02, queue_gate=50)
+    grid = generate_sensitivity_grid(base)
+    assert len(grid) > 10
+    labels = [label for label, _params in grid]
+    assert "Baseline" in labels
+    assert any("offset=" in l for l in labels)
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)